Doc/library/os.rst

```diff
@@ -193,10 +193,6 @@ process and user.
    to the environment made after this time are not reflected in :data:`os.environ`,
    except for changes made by modifying :data:`os.environ` directly.
 
-   The :meth:`!os.environ.refresh` method updates :data:`os.environ` with
```

Comment:
I think that _some_ reference to `reload_environ()` should be left in the `environ` description. Maybe add it to the previous paragraph.

Reply:
I added a "seealso" with a link to the function.